packages/web3/src/utils/sign.ts

```diff
@@ -40,7 +40,7 @@ necc.utils.hmacSha256Sync = (key: Uint8Array, ...messages: Uint8Array[]): Uint8A
 export function sign(hash: string, privateKey: string, _keyType?: KeyType): string {
```

Comment:
This function only handles `secp256k1` and `schnorr`
- either we add support for other signatures
- or we limit the KeyType in the function signature